/crawler_node:
  ros__parameters:
    device_name:          "EPOS4"
    protocol_stack_name:  "MAXON SERIAL V2"
    interface_name:       "USB"
    port_name:            "USB0"
    id_host:              1

    crawler_left:
      id: 1
    crawler_right:
      id: 2
